Jenkinsfile

```diff
@@ -7,8 +7,7 @@ def publishDockerhubICRImages = false
 pipeline {
     agent {
         node {
-            label "ec2-fleet"
-            customWorkspace("/tmp/workspace/${env.BUILD_TAG}")
```

Comment:
Was removing the custom workspace intention? That's pretty much the entire point of this PR?  Also I bet this can just be closed if you  weren't specifically interesting in this PR, and instead merged master to all of them.

Reply:
@jmoses 
this PR has not been merged yet. still in draft?
``` 
customWorkspace("/tmp/workspace/${env.BUILD_TAG}")
```
looks good. lets rebase and merge.